inductiva/fluids/scenarios/heat_sink/heat_sink.py

```diff
@@ -31,6 +31,30 @@ class HeatSink(Scenario):
     The heat source is modeled as a heater with a given power. The heat sink
     is a block of aluminum with thin fins on top. The air flow is introduced
     in the simulation via an inlet, where the air is at a fixed temperature.
+
+    The scenario is simulated in a 3D box with dimensions 8 x 16 x 52 cm in x, y
+    and z directions, respectively. The heat sink is a 4 x 3 x 6 cm block with 1
```

Comment:
I would suggest clarifying the orientation of the fins first and only after mention their separation.